source/main.lua

```diff
@@ -246,11 +246,8 @@ function love.draw()
 	-- this comes BEFORE the TLfres.beginRendering
 	drawWallpaper()
 
-	local strCurrentScreen = fun.CurrentScreenName()

```

Comment:
Isn't this local required to draw stuff properly?

Reply:
I see now. strCurrentScreen has become a global in love.update.

Is that intended? Guessing you want to avoid the same call in love.draw.

Reply:
I removed it because it was shadowing the local variable for main in line 136 and it made do sense redefining the same thing.